Automattic/wp-calypso
# Testing: Remove obsolete useI18n test helper

Fixes #5818

After moving `lib/mixin/i18n` to an external repository, we don't need the i18n test helper anymore.
This PR removes `useI18n` test helper calls.

## Testing Instructions
Check that all the tests still pass: `make test`

Test live: https://calypso.live/?branch=update/remove-use-i18n
